src/components/ContentHighlights/data/hooks.js

```diff
@@ -35,4 +28,58 @@ export const useHighlightSetsForCuration = (enterpriseCuration) => {
   }, [enterpriseCuration]);
 
   return highlightSets;
-};
+}
+
+export function useContentHighlightsContext() {
```

Comment:
nit: We may want to add a JSDOc to this hook to document its intent/purpose, e.g.:

```jsx
/**
 * Defines an interface to mutate the `ContentHighlightsContext` context value.
 */
export function useContentHighlightsContext() {
```

Reply:
[consideration] I wonder if there's a better name for `useContentHighlightsContext` to reflect this intent/purpose.